### ISSUE TITLE
[A11y] Actions component navigation for '@fluentui-contrib/react-chat'

### ISSUE THREAD

**Issue by vhuseinova-msft** [microsoft/fluentui-contrib#272]
Currently, the body of the message components in '@fluentui-contrib/react-chat' are considered tabbable elements. However, when the actions component is specified for my message FluentUI component, it is added at the same level as the body component. This breaks the keyboard navigation hierarchy, as both the actions and message body are on the same level. As a result, actions are accessible with the Tab key instead of the Enter key.

**Steps to Reproduce:**

- Use '@fluentui-contrib/react-chat' to create a chat interface.
- Add actions buttons list
- Specify an actions component at the same level as the message body.
- Attempt to navigate using the keyboard.

**Expected Behavior:** The actions component should be accessible with the Enter key, maintaining a proper keyboard navigation hierarchy.

**Actual Behavior:** The actions component is accessible with the Tab key, disrupting the expected navigation flow

**Comment by vhuseinova-msft** [comment]
It looks like this can be updated by using `...useFocusableGroup({ tabBehavior: 'unlimited' })` and handling Enter key locally. But please let me know if there is a better solution for this.

**Comment by YuanboXue-Amber** [comment]
Following up with the creator to request an example

**Comment by vhuseinova-msft** [comment]
Attaching the POC
https://github.com/vhuseinova-msft/fluentui-contrib-github-issue-272
